testsuite/db-allocator-plugin/pom.xml

```diff
@@ -74,7 +70,9 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-plugin-plugin</artifactId>
-                <version>3.6.0</version>
+                <configuration>
+                    <goalPrefix>db-allocator</goalPrefix>
```

Comment:
Using a newer version of `maven-plugin-plugin` requires us to provide a `goalPrefix` where this was previously inferred. I have specified this based on what is documented under the ["Plugin Prefix Resolution"](https://maven.apache.org/guides/introduction/introduction-to-plugin-prefix-mapping.html) documentation, but I'd like another set of eyes to confirm this is correct.

Incidentally, it looks like we can omit this if we were following the naming [conventions for plugins](https://maven.apache.org/guides/introduction/introduction-to-plugin-prefix-mapping.html#specifying-a-plugins-prefix), which would rename the plugin to `db-allocator-maven-plugin`.

Reply:
If it's the case only for the db-allocator, I would not bother about it as db-allocator will be removed soon. See https://github.com/keycloak/keycloak/pull/30406

Reply:
I see now that the PR you mention was merged and this file no longer exists, so we can consider this resolved.